app/src/main/java/org/greenstand/android/TreeTracker/models/Locations.kt

```diff
@@ -297,19 +317,17 @@ class Convergence(val locations: List<Location>) {
         newValue: Double
     ): ConvergenceStats {
         val newMean = currentStats.mean -
-                        (replacingValue / CONVERGENCE_DATA_SIZE) +
-                        (newValue / CONVERGENCE_DATA_SIZE)
+                        (replacingValue / locations.size) +
```

Comment:
Given Convergence is passed a list of data named `locations`, the size is already fixed. Convergence wouldn't be created until the required convergence data size is reached  by the caller. Hence it makes sense to use the size of `locations` to do the computation and this also eliminates the need to inject configuration into this object primarily used for computation. The caller instantiating this object is responsible for ensuring the correct size of data is passed for computation and Convergence is only concerned with the given data and its computation.